tests/CommunityToolkit.Mvvm.UnitTests/Test_ObservablePropertyAttribute.cs

```diff
@@ -273,6 +273,21 @@ public void Test_AlsoNotifyChangeFor()
         CollectionAssert.AreEqual(new[] { nameof(model.Surname), nameof(model.FullName), nameof(model.Alias) }, propertyNames);
     }
 
+    [TestMethod]
+    public void Test_OnPropertyChangingAndChangedPartialMethods()
```

Comment:
Do we have tests somewhere that check that the OnChanging/OnChanged events fire in the right order and at the right time as setting the property values?

If so, do we want to replicate those here for these methods as well? Just guard against accidentally modifying behavior if code moves/refactors in the future?

Reply:
Good idea! Done in 17c2cf4725d1d20b08e326c358927870f8f4071f, thanks! 😄